package.json

```diff
@@ -40,7 +40,7 @@
     "benchmark": "^2.1.4",
     "chai": "^4.4.1",
     "chalk": "^5.3.0",
-    "dbx-js-tools": "github:mongodb-js/dbx-js-tools",
+    "dbx-js-tools": "github:mongodb-js/dbx-js-tools#NODE-6358",
```

Comment:
Once https://github.com/mongodb-js/dbx-js-tools/pull/20 merges I'll revert this back to main.